pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -53,32 +58,63 @@ func TestCertManager(t *testing.T) {
 		helm.Namespace(meshNamespace).
 			Chart("jetstack/cert-manager-istio-csr").
 			Release("istio-csr").
+			Version("v0.6.0").
 			ValuesString(istioCsrValues(meshNamespace, smcpName)).
 			Install(t)
 		oc.WaitPodsReady(t, meshNamespace, "app=cert-manager-istio-csr")
 
-		t.LogStep("Deploy the cert-manager in SMCP")
-		oc.ApplyString(t, meshNamespace, createSMCPWithCertManager(smcpName, meshNamespace, ns))
-		oc.WaitSMCPReady(t, meshNamespace, smcpName)
-
-		t.LogStep("Install httpbin and sleep")
-		app.InstallAndWaitReady(t, app.Httpbin(ns), app.Sleep(ns))
-
-		t.LogStep("Check if httpbin returns 200 OK ")
-		retry.UntilSuccess(t, func(t test.TestHelper) {
-			oc.Exec(t,
-				pod.MatchingSelector("app=sleep", ns),
-				"sleep",
-				`curl http://httpbin:8000/ip -s -o /dev/null -w "%{http_code}"`,
-				assert.OutputContains(
-					"200",
-					"Got expected 200 OK from httpbin",
-					"Expected 200 OK from httpbin, but got a different HTTP code"))
-		})
+		for _, ver := range []string{"v2.3", "v2.4"} {
```

Comment:
I'm working on a PR that will enable us to run the entire test suite against a specific SMCP version. And the `make test` command will run the test suite for each of the supported versions. So, can you remove this loop and use `env.GetSMCPVersion()` here. You can run the test twice (with the `SMCPVERSION` env var set to a different version each time).